src/main/java/formflow/library/config/TemplateManager.java

```diff
@@ -10,7 +10,6 @@
 import java.lang.reflect.Constructor;
 import java.util.HashMap;
 import java.util.List;
-import java.util.stream.Collectors;
```

Comment:
I took this out as it's unused.

Reply:
Ohh we can tell [IntelliJ to remove unused imports on the fly](https://intellij-support.jetbrains.com/hc/en-us/community/posts/206915775-How-to-enable-remove-unused-imports-#community_comment_207047099).